### ISSUE TITLE
Suggestion to replace Path functions with SHLWAPI

### ISSUE THREAD

**Issue by leeter** [Microsoft/winfile#84]
Given the current windows 7 target we can't use the new path library that properly supports unicode pathing but we can use the [SHLWAPI version](https://msdn.microsoft.com/en-us/library/windows/desktop/bb773559(v=vs.85).aspx) which is patch and "secure" this would allow for the wholesale removal of that code.

**Comment by craigwi** [comment]
Yes, that would be better for newer versions of Windows, but wouldn't that complicate the ver9x branch (when created)?

**Comment by ZanderBrown** [comment]
Yes
